src/styles/app.scss

```diff
@@ -1,9 +1,12 @@
+$mainBackgroundColor: #e1f3f8;
+// @import './settings.scss';
+
 main {
-  padding-bottom: 200px;
-  background-color: #e1f3f8;
+  padding-bottom: 250px;
```

Comment:
Make sure your padding is good for all screen orientations and sizes.

Reply:
Thanks - I was using padding so the footer have an absolute position at the bottom but not be covered by or cover the main content. Maybe I could just use Grid?

Reply:
Try it!

Reply:
Try it! If you remember to, let me know how it turns out!